src/extras/gizmo/gizmo.js

```diff
@@ -33,15 +33,15 @@ const ORTHO_SCALE_RATIO = 0.32;
  * @type {string}
  * @category Gizmo
  */
-export const GIZMO_LOCAL = 'local';
+export const GIZMOCOORD_LOCAL = 'local';
```

Comment:
I would consider moving all constants from few modules here to `constants.js` as we do in the engine to keep consistent. This also allows the import of constants, without importing the rest of the module, which often avoids circular references to be created.